### PR TITLE
chore: add snapshot testing

### PR DESCRIPTION
**Pull request by thiyagaraj-eightfold**
Setting up plugin and adding auto generated snapshot testing via storybook stories

### CODE REVIEW

File: src/storybook.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import initStoryshots from '@storybook/addon-storyshots';
+
+Object.defineProperty(window, 'matchMedia', {
```
Comment (yrchhabra-eightfold) [inline src/storybook.test.js:3]: Can we break the snapshots based on stories, otherwise we'll get into lot of conflicts

**Review by yrchhabra-eightfold — CHANGES_REQUESTED** [review]

**Comment by yrchhabra-eightfold** [comment]
Also change feat to chore in commit message

File: src/storybook.test.js (hunk 1 above)
Comment (thiyagaraj-eightfold) [reply src/storybook.test.js:3]: The snapshots are managed by the plugin, it maintains the order, etc. to avoid the conflicts.

File: yarn.lock [hunk 2]
```diff
@@ -1365,6 +1372,108 @@
...
+  version "10.0.29"
+  resolved "https://registry.yarnpkg.com/@emotion/cache/-/cache-10.0.29.tgz#87e7e64f412c060102d589fe7c6dc042e6f9d1e0"
+  integrity sha512-fU2VtSVlHiF27empSbxi1O2JFdNWZO+2NFHfwO0pxgTep6Xa3uGb+3pVKfLww2l/IBGLNEZl5Xf/++A4wAYDYQ==
+  dependencies:
+    "@emotion/sheet" "0.9.4"
+    "@emotion/stylis" "0.8.5"
+    "@emotion/utils" "0.11.3"
+    "@emotion/weak-memoize" "0.2.5"
+
+"@emotion/core@^10.1.1":
```
Comment (yrchhabra-eightfold) [inline yarn.lock:1385]: why is emotion here ?

**Review by yrchhabra-eightfold — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (thiyagaraj-eightfold) [reply yarn.lock:1385]: Not sure, regenerating the lock file

Comment (thiyagaraj-eightfold) [reply yarn.lock:1385]: Its a dependency, of the storyshots plugin

**Review by yrchhabra-eightfold — APPROVED** [review]

**Pull request merged**